example_test.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials_test
+
+import (
+	"context"
+	"fmt"
+
+	credentials "github.com/oras-project/oras-credentials-go"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func ExampleNewStoreFromDocker() {
+	// NewStoreFromDocker returns a Store based on the default docker config file.
+	// See the documentation for details.
+	ds, err := credentials.NewStoreFromDocker(credentials.StoreOptions{AllowPlaintextPut: true})
```

Comment:
Maybe we don't need these comments here.
![image](https://user-images.githubusercontent.com/8113721/234780626-6491bbdf-0fa1-40f0-aa89-786a48df4be7.png)

Reply:
deleted